### PR TITLE
Gpl769 change encoding

### PR DESCRIPTION
**Pull request by emrojo**
Closes GPL679

Changes proposed in this pull request:

* Change encoding
* Create the view after changing the encoding

### CODE REVIEW

File: .github/workflows/docker_ci.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
           RABBITMQ_DEFAULT_USER: guest
           RABBITMQ_DEFAULT_PASS: guest
       ewh:
-        image: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:master
+        image: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:development
```
Comment (pjvv) [inline .github/workflows/docker_ci.yml:39]: Don't we want this as master?

File: .github/workflows/docker_ci.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
           RMQHOST: rmq
           INTEGRATION_TEST_SETUP: true
           INTEGRATION_TEST_SEED: '/code/spec/data/integration/seed_for_unified_wh.rb'
-          RAILS_ENV: test
```
Comment (pjvv) [inline .github/workflows/docker_ci.yml:48]: And this as test?

File: .github/workflows/docker_ci.yml (hunk 1 above)
Comment (emrojo) [reply .github/workflows/docker_ci.yml:39]: yes, but it wont work until is merged into master

File: .github/workflows/docker_ci.yml (hunk 2 above)
Comment (emrojo) [reply .github/workflows/docker_ci.yml:48]: This is the issue with views_schema in unified warehouse. 

Views schema generates a dump of the view definition from the local database, and this definition is what we use for local development and for testing. The problem is that the view definition refers to a specific database (either development or test) when we are joining with event warehouse; I choose the option of generating the view from development as the main reason we want the definition is for development. The problem we have is that then the view is created using development databases for event warehouse, so then we need to create the development database for event warehouse.

**Review by pjvv — APPROVED** [review]

**Pull request merged**